src/utilcode/util.cpp

```diff
@@ -1288,7 +1288,15 @@ int GetCurrentProcessCpuCount()
     // the case where there are more than 64 processors, so we will return a
```

Comment:
@Maoni0 the GetCurrentProcessCpuCount is called only when the CPU groups are not enabled. And this function returns max 64 on Windows too. I think that without CPU groups enabled, the 64 is still the limit for this specific function. For CPU groups, GC code uses CPUGroupInfo::GetNumActiveProcessors() instead to get the cpu count. That will work for Unix too later.
I have recently added NUMA support for Unix, including the CPU groups, but it is not enabled yet since I didn't have enough time to fully test it in the 2.0 timeframe, especially on machines with a lot of processors. I have also decided later to make it runtime optional based on whether the libnuma is installed or not.